admin/release

```diff
@@ -0,0 +1,162 @@
+ERROR='\033[0;31m'
+
+echo -e "${BOLD}${COLOR}CodeIgniter4 release builder${NORMAL}"
+echo '----------------------------'
+
+#---------------------------------------------------
+# Check arguments
+echo -e "${BOLD}Checking arguments...${NORMAL}"
+
+if [ $# -lt 1 ]; then
+    echo "You really need to read the directions first!"
+    exit 1
+fi
+
+if [ $1 = 'deploy' ]; then
+    action=deploy
+elif [ $1 != 'test' ]; then
+    echo -e "${ERROR}Invalid action ($1)${NORMAL}"
+    exit 1
+fi
+shift   # drop the command from the argument list
+
+version=$1
+if [ $# -gt 1 ]; then
+    qualifier="-${2}"
+    which='pre-release'
+fi
+branch="release-$version$qualifier"
+
+#---------------------------------------------------
+# Create the release branch
+echo -e "${BOLD}Creating $which $branch to $action ${NORMAL}"
+
+git checkout develop
+git branch -d $branch     # remove the branch if there
+git checkout -b $branch
+
+#---------------------------------------------------
+# Update version dependencies
+echo -e "${BOLD}Updating version dependencies${NORMAL}"
+
+# CI_VERSION definition in system/CodeIgniter.php
+sed -i "/const CI_VERSION/s/'.*'/'${version}${qualifier}'/" system/CodeIgniter.php
+
+# release substitution variable in user_guide_src/source/conf.py
+sed -i "/release =/s/'.*'/'${version}${qualifier}'/" user_guide_src/source/conf.py
+
+# version & date in user_guide_src/source/changelog.rst
+sed -i "/|release|/s/|.*|/${version}${qualifier}/" user_guide_src/source/changelog.rst
+sed -i "/Release Date/s/Not Released/$(date +'%B %d, %Y')/" user_guide_src/source/changelog.rst
+
+#---------------------------------------------------
+# Generate the user guide
+echo -e "${BOLD}Generate the user guide${NORMAL}"
+cd user_guide_src
+
+# make the UG & embed it in the project root
+make html
+rm -Rf ../UserGuide
+mv build/html ../UserGuide
+
+# make the epub & stick it in the project's "build" folder for now
+make epub
+mv build/epub/CodeIgniter4.epub "../build/CodeIgniter-UserGuide-${version}${qualifier}.epub"
+
+cd ..
+
+#---------------------------------------------------
+# Hide stuff from the release bundle
+echo -e "${BOLD}Hide stuff from the release bundle${NORMAL}"
+
+# Restore the old .gitignore 
+if [ -f admin/previous-gitignore ]; then
+    cp -r admin/previous-gitignore .gitignore
+fi
+cp -r .gitignore admin/previous-gitignore
+
+# Add the dev only folders/files to .gitignore
+for f in $devonly; do
+    echo $f >> .gitignore
+done
+
+#---------------------------------------------------
+# And finally, the release tag
+echo -e "${BOLD}Tag this branch for release${NORMAL}"
+git tag -a v${version}${qualifier} -m "Release ${version}${qualifier} build"
+
+#---------------------------------------------------
+# Are we there yet?
+if [ $action = 'test' ]; then
+    echo -e "${BOLD}Your $branch branch is ready to inspect.${NORMAL}"
+    exit 0
+fi
+echo -e "${BOLD}Are we there yet (yes|no)?${NORMAL}"
+read answer
+if [ $answer != 'yes' ]; then
```

Comment:
I see no exit/return inside this check ... script execution would continue.

Reply:
Doh - you are right! Thank you for pointing that out :)

Reply:
Fixed it :)